h2o-core/src/main/java/water/H2O.java

```diff
@@ -1324,6 +1324,9 @@ public static String DEFAULT_FLOW_DIR() {
    * It is updated also when a new client appears. */
   private static HashSet<H2ONode> STATIC_H2OS = null;
```

Comment:
`STATIC_H2OS` are still accessed as before, access to this field wasn't changed, however the `CLIENTS` are field is accessed from multiple threads